src/org/usfirst/frc/team2706/robot/subsystems/Camera.java

```diff
@@ -15,178 +14,31 @@
  * Does everything java related to the rPI camera, servo controllers have been deleted.
  */
 public class Camera extends Subsystem {
-
+    // TODO we should probably consolidate these two IP addresses
+    public static final String RPI_IP = "10.27.6.240";
```

Comment:
I was thinking, maybe this should go into RobotMap? Theres a small change the RPI ips are different on the practice and competition bots, so RobotMap could have all of the values in one location